### PR TITLE
PS-4692 Add paragraph for how to manually install/uninstall MyRocks

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged with 5.7 version

### CODE REVIEW

**Review by nastena1606 — APPROVED** [review]
LGTM

File: doc/source/myrocks/install.rst [hunk 1]
```diff
@@ -96,6 +96,28 @@ Alternatively, you can add ``ENGINE=RocksDB``
...
+    INSTALL PLUGIN ROCKSDB_PERF_CONTEXT SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_PERF_CONTEXT_GLOBAL SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_CF_OPTIONS SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_GLOBAL_INFO SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_COMPACTION_STATS SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_DDL SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_INDEX_FILE_MAP SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_LOCKS SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_TRX SONAME 'ha_rocksdb.so';
+    INSTALL PLUGIN ROCKSDB_DEADLOCK SONAME 'ha_rocksdb.so';
```
Comment (inikep) [inline doc/source/myrocks/install.rst:121]: There are more plugins now: 
```
INSTALL PLUGIN ROCKSDB SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_CFSTATS SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_DBSTATS SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_PERF_CONTEXT SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_PERF_CONTEXT_GLOBAL SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_CF_OPTIONS SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_GLOBAL_INFO SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_COMPACTION_HISTORY SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_COMPACTION_STATS SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_ACTIVE_COMPACTION_STATS SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_DDL SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_INDEX_FILE_MAP SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_LOCKS SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_TRX SONAME 'ha_rocksdb.so';
INSTALL PLUGIN ROCKSDB_DEADLOCK SONAME 'ha_rocksdb.so';
```

File: doc/source/myrocks/install.rst [hunk 2]
```diff
@@ -136,4 +158,26 @@ Finally, remove all the :ref:`myrocks_server_variables`
...
+    UNINSTALL PLUGIN ROCKSDB_PERF_CONTEXT;
+    UNINSTALL PLUGIN ROCKSDB_PERF_CONTEXT_GLOBAL;
+    UNINSTALL PLUGIN ROCKSDB_CF_OPTIONS;
+    UNINSTALL PLUGIN ROCKSDB_GLOBAL_INFO;
+    UNINSTALL PLUGIN ROCKSDB_COMPACTION_STATS;
+    UNINSTALL PLUGIN ROCKSDB_DDL;
+    UNINSTALL PLUGIN ROCKSDB_INDEX_FILE_MAP;
+    UNINSTALL PLUGIN ROCKSDB_LOCKS;
+    UNINSTALL PLUGIN ROCKSDB_TRX;
+    UNINSTALL PLUGIN ROCKSDB_DEADLOCK;
```
Comment (inikep) [inline doc/source/myrocks/install.rst:185]: Also these:
```
ROCKSDB_COMPACTION_HISTORY
ROCKSDB_ACTIVE_COMPACTION_STATS
```

File: doc/source/myrocks/install.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/myrocks/install.rst:121]: Updated.

File: doc/source/myrocks/install.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/myrocks/install.rst:185]: Updated.

**Review by inikep — APPROVED** [review]
LGTM

**Pull request merged**